src/components/DocumentationTopic/PrimaryContent/DeclarationSource.vue

```diff
@@ -52,6 +51,111 @@ export default {
       required: false,
     },
   },
+  computed: {
+    formattedTokens: ({
+      language,
+      formattedSwiftTokens,
+      tokens,
+    }) => (language === Language.swift.key.api ? formattedSwiftTokens : tokens),
+    // Return a formatted version of the tokens array, with additional
+    // indentation whitespace to break parameters onto individual lines for
+    // improved readability and scanning of Swift functions/initializers.
+    //
+    // This is implemented in a single pass loop where text tokens are updated
+    // at certain key points to insert spaces and newlines so that each
+    // parameter of a multi-parameter function gets its own line.
+    //
+    // Ideally this should be implemented with a tool like SwiftFormat in the
+    // future with the raw text before it gets tokenized for syntax
+    // highlighting—however, this post-tokenization JavaScript logic should be
+    // an improvement until that kind of work can be integrated.
+    //
+    // @param {Array} tokens The original syntax tokens.
+    //   See `DeclarationToken.props`
+    // @return {Array} A formatted version of the original tokens.
+    //   See `DeclarationToken.props`
+    formattedSwiftTokens: ({ tokens }) => {
```

Comment:
Hm, we can do that, although it seems a little unnecessary since this should only be called (and only is called today) when the language is Swift. Technically, if it _were_ to be called in this situation, the code would run, but the tokens would be returned unchanged as expected since none of the triggering conditions would apply to realistic Objective-C symbol declarations.

Reply:
Alright, leave it as is.